.ci/after_success.sh

```diff
@@ -4,8 +4,10 @@ set -ev
 
 bash <(curl -s https://codecov.io/bash)
 
+AVALANCHE_IMAGE=$(docker image ls --format="{{.Repository}}:{{.Tag}}" | grep $DOCKERHUB_REPO | head -n 1)
+
 TRAVIS_TAG="$DOCKERHUB_REPO:travis-$TRAVIS_BUILD_NUMBER"
-docker tag $DOCKERHUB_REPO:$COMMIT "$TRAVIS_TAG"
+docker tag $AVALANCHE_IMAGE "$TRAVIS_TAG"
```

Comment:
Previously we were tagging the commit here, is the `.Tag` variable set to the commit?

Reply:
The .Tag variable is the tag of the docker image built by `scripts/build_image.sh`. So it's of the form <repository>:<tag> ie. avaplatform/avalanchego:latest

Reply:
Changed this by addressing the difference in length of the commit hashes instead of using this line. It looks like there's some cross-platform differences with the line `git rev-pars --short HEAD`, it creates an 8 character string on my mac, but within CI only uses the first 7 characters, so I updated the way this was used in the build image scripts to explicitly take the first 8 characters as is done with converting `TRAVIS_COMMIT` into the environment variable `COMMIT` within `.travis.yml`